src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -490,90 +448,85 @@ internal SynthesizedAttributeData SynthesizeDebuggerStepThroughAttribute()
             return TrySynthesizeAttribute(WellKnownMember.System_Diagnostics_DebuggerStepThroughAttribute__ctor);
         }
 
-        internal void EnsureIsReadOnlyAttributeExists(DiagnosticBag diagnostics, Location location, bool modifyCompilation)
+        private void EnsureEmbeddedAttributeExists(EmbeddedAttributes attribute, DiagnosticBag diagnostics, Location location, bool modifyCompilation)
```

Comment:
>EnsureEmbeddedAttributeExists [](start = 21, length = 29)

The name is confusing:
- We have an EmbeddedAttribute
- The attributes are not always embedded.

Consider renaming to something like "EnsureEmbedd**able**AttributeExists"